### PR TITLE
fix: blurry qr code when converting svg template to pdf certificate

### PR DESCRIPTION
**Pull request by varadeth**

### CODE REVIEW

File: services/certificate-api/main.js [hunk 1]
```diff
@@ -1,17 +1,25 @@
...
     console.time(label)
     console.log(`API ${req.method} ${req.url} called`);
+    let isPipedResponse = false;
     try {
         if (req.method === 'GET' && req.url.startsWith("/health")) {
             res.end("OK")
         } else if (req.method === 'POST' && req.url.startsWith("/api/v1/certificate") && ["application/pdf"].includes(req.headers.accept)) {
             const data = await certificateController.getCertificatePDF(req, res);
-            res.end(data)
+            isPipedResponse = data instanceof PDFDocument
```
Comment (tejash-jl) [inline services/certificate-api/main.js:16]: getCertificatePDF can handle this right?

File: services/certificate-api/src/routes/certificate_controller.js [hunk 2]
```diff
@@ -247,6 +248,14 @@ async function renderDataToTemplate(templateFileURL, data) {
 
 async function createPDF(certificate) {
     try {
+        if(certificate.indexOf("svg") !== -1) {
+            const doc = new PDFDocument({
+                layout: "portrait",
```
Comment (tejash-jl) [inline services/certificate-api/src/routes/certificate_controller.js:253]: is the layout mandatory? what if it is landscape?

Comment (varadeth) [reply services/certificate-api/src/routes/certificate_controller.js]: it is not mandatory but by default the value is portrait and A4

**Comment by tejash-jl** [comment]
Closing the PR as it's not relevant anymore.

**Pull request closed without merging**